backend/src/polling_station/structs.rs

```diff
@@ -510,27 +764,234 @@ mod tests {
             &mut validation_results,
             "polling_station_results".to_string(),
         );
-        assert_eq!(validation_results.errors.len(), 2);
+        assert_eq!(validation_results.errors.len(), 3);
+        assert_eq!(validation_results.warnings.len(), 0);
+        assert_eq!(
+            validation_results.errors[0].code,
+            ValidationResultCode::IncorrectTotal
+        );
+        assert_eq!(
+            validation_results.errors[1].code,
+            ValidationResultCode::IncorrectTotal
+        );
+        assert_eq!(
+            validation_results.errors[2].code,
+            ValidationResultCode::IncorrectDifference
+        );
+
+        // test F.11 incorrect total, F.12 incorrect total, F.13 incorrect total and F.22 incorrect difference
+        validation_results = ValidationResults::default();
+        let polling_station_results = PollingStationResults {
+            recounted: true,
+            voters_counts: VotersCounts {
+                poll_card_count: 1,
+                proxy_certificate_count: 2,
+                voter_card_count: 3,
+                total_admitted_voters_count: 5, // F.11 incorrect total
+            },
+            votes_counts: VotesCounts {
+                votes_candidates_counts: 3,
+                blank_votes_count: 2,
+                invalid_votes_count: 1,
+                total_votes_cast_count: 5, // F.12 incorrect total
+            },
+            voters_recounts: Some(VotersRecounts {
+                poll_card_recount: 2,
+                proxy_certificate_recount: 1,
+                voter_card_recount: 4,
+                total_admitted_voters_recount: 8, // F.13 incorrect total
+            }),
+            differences_counts: DifferencesCounts {
+                more_ballots_count: 0,
+                fewer_ballots_count: 0, // F.22 incorrect difference
+                unreturned_ballots_count: 0,
+                too_few_ballots_handed_out_count: 0,
+                too_many_ballots_handed_out_count: 0,
+                other_explanation_count: 0,
+                no_explanation_count: 0,
+            },
+            political_group_votes: vec![PoliticalGroupVotes {
+                number: 1,
+                total: 3,
+                candidate_votes: vec![CandidateVotes {
+                    number: 1,
+                    votes: 3,
+                }],
+            }],
+        };
+        let election = election_fixture(&[1]);
+        polling_station_results.validate(
+            &election,
+            &mut validation_results,
+            "polling_station_results".to_string(),
+        );
+        assert_eq!(validation_results.errors.len(), 4);
         assert_eq!(validation_results.warnings.len(), 0);
+        // TODO: Is showing F.11 Incorrect Total still wanted if there was a recount?
```

Comment:
@praseodym @jorisleker If there was a recount, what do we do with a potential error in the original voters count. I'd say only show the error in voters recount if there is one, but the current implementation will show both and will be blocking.

Reply:
Good point! I think it's weird to block on this (the recount has been done for a reason) but that brings up the question why the original counts need to be entered at all. @jorisleker do you know more about this?

Reply:
Aha good question, I'd say to reflect there actually was a recount, if this is actually showing up in the PV though...

Reply:
I have now changed the code to validate voters_counts if not recouted and to validate voters_recounts if recounted.

Reply:
As discussed with @arjen-ba (and @jschuurk-kr and @marjoleintamis) in the standup:
- A to D (rubriek 1) should not validated if A.2 to D.2 (rubriek 3) is filled in. (Sometimes the original numbers A to C don't even add up to D).
- To make the data entry process easier, it might be easier to follow the order of the original PV and also enter the values for A to D into Abacus when A.2 to D.2 is filled in (@jorisleker will have an opinion on this).
- The numbers A to D are not used when A.2 to D.2 are filled in. However, they might be useful in the EML file to be used for data analysis.